package.json

```diff
@@ -4,6 +4,8 @@
   "private": true,
   "dependencies": {
     "@material-ui/core": "^1.2.1",
+    "axios": "^0.18.0",
+    "eslint": "^4.19.1",
```

Comment:
Probably `eslint-plugin-react` package is also required.
I couldn't work eslint without adding this.